src/lib.rs

```diff
@@ -9,9 +9,10 @@ use libc::ioctl;
 use std::error::Error;
```

Comment:
Could you give an example where you would want to read the frame? You have previously written it yourself, so why not keep this data stored somewhere?

Reply:
I have made my own display that is using SPI and I want to read the framebuffer and push it over SPI to the display. In my case the OS/applications are writing to the framebuffer and I'm just pushing it to the display.

Reply:
I want to make screenshot of the framebuffer (drawn by an another application), so I need this logic too.